api/client/src/main/java/org/projectnessie/client/rest/ResponseCheckFilter.java

```diff
@@ -113,14 +114,23 @@ private static NessieError decodeErrorObject(
         JsonNode errorData = reader.readTree(capturing);
         try {
           error = reader.treeToValue(errorData, NessieError.class);
+        } catch (MismatchedInputException e) {
```

Comment:
Is that possible with the example from the description? `NessieNotAuthorizedException: Unauthorized (HTTP/401)` does not look like valid JSON, so I'd guess parsing would fail on line 114 before we try to match JSON to java types.

Reply:
bogus comment, please ignore